system/Autoloader/Autoloader.php

```diff
@@ -221,45 +205,57 @@ public function removeNamespace(string $namespace)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Loads the class file for a given class name.
+	 * Loads the class file for the given class name.
 	 *
 	 * @param string $class The fully qualified class name.
 	 *
-	 * @return string|false The mapped file on success, or boolean false
-	 *                          on failure.
+	 * @return string|false The mapped file on success, or false on failure.
 	 */
 	public function loadClass(string $class)
 	{
-		$class = trim($class, '\\');
-		$class = str_ireplace('.php', '', $class);
+		$class = str_ireplace('.php', '', trim($class, '\\'));
 
-		$mapped_file = $this->loadInNamespace($class);
+		$mappedFile = $this->loadInNamespace($class);
```

Comment:
use underscore on purpose as removed in 4.1, not worth updated for conflict, see rector.php config to skip this file for check underscore to camelcase check